Firestore/Protos/nanopb_pretty_printing/generators.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python
```

Comment:
How about `lib/pretty_printing.py` as the filename for this?

I can imagine we'll have other things we can put in `lib`, but it's doubtful we'll put anything in `nanopb_pretty_printing`.

Note that all we're trying to do is disambiguate top-level scripts from the parts they use. I doubt we'd have any folder structure underneath `lib` unless the amount of code generation goes through the roof.